CHANGELOG.md

```diff
@@ -34,6 +29,16 @@ The types of changes are:
 ### Docs
 - Added docs for PrivacyNoticeRegion type [#5488](https://github.com/ethyca/fides/pull/5488)
 
+## [2.49.1](https://github.com/ethyca/fidesplus/compare/2.49.0...2.49.1)
+
+### Added
+- Added support for GPP national string to be used alongside state-by-state using a new approach option [#5480](https://github.com/ethyca/fides/pull/5480)
+- Added "Powered by" branding link to privacy center and Layer 2 CMP [#5483](https://github.com/ethyca/fides/pull/5483)
+- Added loading state to the toggle switches on the Manage privacy notices page [#5489](https://github.com/ethyca/fides/pull/5489)
+
+### Fixed
+- Use BlueConic Profile API correctly. [#5487](https://github.com/ethyca/fides/pull/5487)
```

Comment:
This was just the quick fix for the real work, which Added BlueConic support to FidesJS
https://github.com/ethyca/fides/pull/5479

Reply:
added the init PR. Thank you!